pymongo/message.py

```diff
@@ -1061,7 +1061,7 @@ def write_command(
             reply = self.conn.write_command(request_id, msg, self.codec)
             if self.publish:
                 duration = (datetime.datetime.now() - start) + duration
-                self._succeed(request_id, reply, duration)
+                self._succeed(request_id, reply, duration)  # type: ignore[arg-type]
```

Comment:
Why ignore arg-type here?

Reply:
yes! that makes a difference, just fixed the types here